### DISCUSSION TITLE
Optimal number of peak calling rounds

### DISCUSSION THREAD

**Discussion by paulasoler** [stuart-lab/signac#675] [Q&A] [answered]
Hello @timoast !
I am doing a multi-level clustering approach to define the different cell-types that we can find in our scATAC dataset. Imagine that we first define level 1 with the main cell-types as B-cell and T-cells, then within the T-cells we re-cluster the data to distinguish the main T-cells sub-types and within each subgroup another clustering to define smaller groups and progressively...would it be interesting to do a new peak calling round in each level of the clustering?. I hope that you have understood my point.

Thanks a lot.

**Comment by timoast** [comment, marked as answer]
You could certainly try it and see whether you start to detect new peaks that were unable to be detected in the first round of cluster-specific peak calling. It's often useful to explore the data interactively using the `CoverageBrowser()` function, and you should be able to see whether you have distinct peaks that were missed by the peak caller.
